From: Alexander Larsson <alexl@redhat.com>
Date: Tue, 17 Jan 2017 12:39:36 +0100
Subject: demos/bubblewrap-shell.sh: Unshare all namespaces

Lets demo things as sandboxed as possible.

Closes: #154
Approved by: cgwalters

Origin: upstream, 0.1.7, commit:a6e15164f94cc95964ff9cd2951e887dfb285174
---
 demos/bubblewrap-shell.sh | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/demos/bubblewrap-shell.sh b/demos/bubblewrap-shell.sh
index 96b87b6..9fccbd7 100755
--- a/demos/bubblewrap-shell.sh
+++ b/demos/bubblewrap-shell.sh
@@ -19,6 +19,11 @@ set -euo pipefail
       --symlink usr/sbin /sbin \
       --chdir / \
       --unshare-pid \
+      --unshare-user-try \
+      --unshare-ipc \
+      --unshare-net \
+      --unshare-uts \
+      --unshare-cgroup-try \
       --dir /run/user/$(id -u) \
       --setenv XDG_RUNTIME_DIR "/run/user/`id -u`" \
       --setenv PS1 "bwrap-demo$ " \
